recipe/meta.yaml

```diff
@@ -35,6 +35,9 @@ requirements:
     - {{ compiler('cxx') }}
     - {{ compiler('cuda') }}
     - sysroot_linux-64 2.17  # [linux64]
+  host:
+    # https://github.com/NixOS/patchelf/issues/492
+    - patchelf !=0.18
```

Comment:
This is a nicer solution! Have so far taken the heavy hammer approach of just [disabling `binary_relocation`]( https://github.com/conda-forge/cudnn-feedstock/commit/8404440b86314a3f4f3ede31ff9db7d09b1f8612 )

Reply:
Yes that's why I asked about it in https://github.com/conda-forge/conda-smithy/issues/1737. I think setting `binary_relocation` is a bit error prone & tedious.

Reply:
Also, there could be bugs uncaught in feedstocks that do not do sanity checks against binaries (like we do here) due to using patchelf 0.18, although admittedly there aren't many feedstocks doing binary repackaging.